res/css/_common.scss

```diff
@@ -381,6 +391,7 @@ textarea {
 .mx_Dialog button.mx_Dialog_primary, .mx_Dialog  input[type="submit"].mx_Dialog_primary {
     color: $accent-fg-color;
     background-color: $accent-color;
+    min-width: 156px;
```

Comment:
Where does that size come from?

Reply:
Just taken from the zeplin from what the size of the button was